packages/db/src/controller/level.ts

```diff
@@ -175,6 +177,17 @@ export class LevelDbController implements IDatabaseController<Uint8Array, Uint8A
 
     this.metrics?.dbWriteItems.inc({bucket}, itemsRead);
   }
+
+  /** Add collect function to capture metric for db size */
+  private addDbSizeMetricCollector(metrics: ILevelDbControllerMetrics): void {
+    const minKey = Buffer.from([0x00]);
+    const maxKey = Buffer.from([0xff]);
+
+    metrics.dbSizeTotal.addCollect(async () => {
+      const dbSize = await this.approximateSize(minKey, maxKey);
```

Comment:
Should this be this.db.approximateSize ?

Reply:
I could call `this.db.approximateSize` but there is a wrapper method in the class itself which restrict start and end key to be of type `Uint8Array`. 

https://github.com/ChainSafe/lodestar/blob/cbb761900ab1858b8227e6cec2f5e4cd6abb150d/packages/db/src/controller/level.ts#L141-L143

Shouldn't really matter which method of those two we call, should it be changed to `this.db.approximateSize`?

Reply:
ah, yeah my bad, didn't see this